x-pack/platform/plugins/shared/ml/common/util/anomaly_description.ts

```diff
@@ -8,7 +8,10 @@
 import { i18n } from '@kbn/i18n';
 import { type MlAnomaliesTableRecordExtended } from '@kbn/ml-anomaly-utils';
 
-export function getAnomalyDescription(anomaly: MlAnomaliesTableRecordExtended): {
+export function getAnomalyDescription(
+  anomaly: MlAnomaliesTableRecordExtended,
+  escapeUrlLikeStrings: boolean = false
```

Comment:
Not sure about the name, as we don't escape characters. 
- `appendFieldNamePostfix` or `breakAutoLinkifyFieldName` sounds more descriptive. Also worth adding a comment why we're doing that. 
- having `options` as a second argument to accommodate it is easier to extend later 
```suggestion
  options: { breakAutoLinkifyFieldName: boolean } = { breakAutoLinkifyFieldName: false }
```

Reply:
Done in https://github.com/elastic/kibana/pull/226849/commits/03c7be8bf9a19cfcb595287541e645e764b3fb40